tests/topotests/static_routing_with_ebgp/test_static_routes_topo3_ebgp.py

```diff
@@ -0,0 +1,990 @@
+        )
+
+    result = verify_rib(
+        tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+    )
+    assert result is True, "Testcase {} : Failed \nError: Routes are"
+    " missing in RIB".format(tc_name)
+
+    step("Shut nexthop interfaces N1 to N8 one by one")
+    for addr_type in ADDR_TYPES:
+        for nhp in range(0, 8):
+            intf = topo["routers"]["r2"]["links"]["r1-link" + str(nhp)]["interface"]
+            shutdown_bringup_interface(tgen, dut, intf, False)
+
+    result = verify_rib(
+        tgen,
+        addr_type,
+        dut,
+        input_dict_4,
+        next_hop=nh,
+        protocol=protocol,
+        expected=False,
+    )
+    assert result is not True, "Testcase {} : Failed \n"
+    "Error: Routes are still present in RIB".format(tc_name)
+
+    step("No shut the nexthop interfaces N1 to N8 one by one .")
+
+    for addr_type in ADDR_TYPES:
+        for nhp in range(0, 8):
+            intf = topo["routers"]["r2"]["links"]["r1-link" + str(nhp)]["interface"]
+            shutdown_bringup_interface(tgen, dut, intf, True)
+
+    result = verify_rib(
+        tgen, addr_type, dut, input_dict_4, next_hop=nh, protocol=protocol
+    )
+    assert result is True, "Testcase {} : Failed \n"
+    "Error: Routes are missing in RIB".format(tc_name)
+
+    step("Random shut of the nexthop interfaces")
+    randnum = random.randint(0, 7)
+    for addr_type in ADDR_TYPES:
+        intf = topo["routers"]["r2"]["links"]["r1-link" + str(randnum)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, False)
+        nhip = NEXT_HOP_IP["nh" + str(randnum + 1)][addr_type]
+        input_dict_5 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": PREFIX1[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh" + str(randnum + 1)][addr_type],
+                    }
+                ]
+            }
+        }
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            input_dict_5,
+            next_hop=nhip,
+            protocol=protocol,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \n"
+        "Error: Routes are still present in RIB".format(tc_name)
+
+    step("Random no shut of the nexthop interfaces")
+    for addr_type in ADDR_TYPES:
+        intf = topo["routers"]["r2"]["links"]["r1-link" + str(randnum)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, True)
+        nhip = NEXT_HOP_IP["nh" + str(randnum + 1)][addr_type]
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_5, next_hop=nhip, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \n"
+        "Error: Routes are missing in RIB".format(tc_name)
+
+    step("Reload the FRR router")
+    # stop/start -> restart FRR router and verify
+    stop_router(tgen, "r2")
+    sleep(5)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.